skylog/storage/gcp/gcppb/gen.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package gcppb contains proto messages for GCP-based Skylog storage.
+package gcppb
+
+//go:generate protoc -I=. --go_out=$GOPATH/src gcp.proto
```

Comment:
Do this need new `go list` stuff because of gomodules?

Reply:
Not sure how to use it here. Could you elaborate?

At this point I do it the same way as in other proto files in this repo. Perhaps there should be a separate PR fixing all protos.

Reply:
@gdbelvin WDYT?

Reply:
Btw, https://github.com/golang/protobuf/issues/692

Reply:
This is consistent with the other proto generation methods, so this LGTM. 
However, the issue you've linked to does suggest a better way.  Could you experiment with using `paths=source_relative` ?  That would allow us to truly remove any references to $GOPATH.